spec.bs

```diff
@@ -543,6 +556,260 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
+[=fencedframetype/fenced frame reporting map=] |reportingMap| and [=leading bid info=]
+|leadingBidInfo|:
+1. Let |buyerDone|, |sellerDone|, and |componentSellerDone| be [=booleans=], initially false.
+1. If |leadingBidInfo|'s [=leading bid info/component seller=] is null, set |componentSellerDone|
+   to true.
+1. [=iteration/While=]:
+   1. If |buyerDone|, |sellerDone|, and |componentSellerDone| are all true, then
+      [=iteration/break=].
+   1. Wait until |leadingBidInfo|'s [=leading bid info/buyer reporting result=] is not null,
+      or |leadingBidInfo|'s [=leading bid info/seller reporting result=] is not null, or
+      |leadingBidInfo|'s [=leading bid info/component seller reporting result=] is not null.
+   1. If |buyerDone| is false and |leadingBidInfo|'s [=leading bid info/buyer reporting result=]
+      is not null:
+      1. Let |buyerMap| be |leadingBidInfo|'s [=leading bid info/buyer reporting result=]'s
+         [=reporting result/reporting beacon map=].
+      1. If |buyerMap| is null, set |buyerMap| to an empty [=map=] «[]».
+      1. [=Finalize a reporting destination=] with |reportingMap|,
+         {{FenceReportingDestination/buyer}}, and |buyerMap|.
+      1. TODO: Fetch |leadingBidInfo|'s [=leading bid info/buyer reporting result=]'s
+         [=reporting result/report url=].
+      1. Set |buyerDone| to true.
+   1. If |sellerDone| is false and |leadingBidInfo|'s [=leading bid info/seller reporting result=]
+      is not null:
+      1. Let |sellerMap| be |leadingBidInfo|'s [=leading bid info/seller reporting result=]'s
+         [=reporting result/reporting beacon map=].
+      1. If |sellerMap| is null, set |sellerMap| to an empty [=map=] «[]».
+      1. [=Finalize a reporting destination=] with |reportingMap|,
+         {{FenceReportingDestination/seller}}, and |sellerMap|.
+      1. TODO: Fetch |leadingBidInfo|'s [=leading bid info/seller reporting result=]'s
+         [=reporting result/report url=].
+      1. Set |sellerDone| to true.
+   1. If |componentSellerDone| is false and |leadingBidInfo|'s
+      [=leading bid info/component seller reporting result=] is not null:
+      1. Let |componentSellerMap| be |leadingBidInfo|'s
+         [=leading bid info/component seller reporting result=]'s
+         [=reporting result/reporting beacon map=].
+      1. If |componentSellerMap| is null, set |componentSellerMap| to an empty [=map=] «[]».
+      1. [=Finalize a reporting destination=] with |reportingMap|,
+         {{FenceReportingDestination/component-seller}}, and |componentSellerMap|.
+      1. TODO: Fetch |leadingBidInfo|'s [=leading bid info/component seller reporting result=]'s
+         [=reporting result/report url=].
+      1. Set |componentSellerDone| to true.
+
+</div>
+
+<div algorithm="create nested configs">
+
+To <dfn>create nested configs</dfn> given [=generated bid/ad component descriptors=]
+|adComponentDescriptors|:
+1. Let |nestedConfigs| be an empty [=list=] «».

```

Comment:
nit: no need for the `«»` here, `empty [=list=]` is sufficient. Doesn't really matter either way though, just figured I'd let you know.

(here and elsewhere below).

Reply:
Might as well keep it. I like literals.